src/modules/database/database.ts

```diff
@@ -0,0 +1,41 @@
+import mongoose from 'mongoose';
+
+class Database {
+  private static instace: Database | null = null;
+
+  private mongodb = mongoose;
+
+  private uri = 'mongodb://127.0.0.1:27017/local';
+
+  private constructor() {
+    this.dropDatabase();
```

Comment:
Every time that you create this Class instance you drop your mongo db, is that correct ?

Reply:
For now, it is. But further on the project this can be changed.